### PR TITLE
Add LEDU Token (BEP20)

### PR DESCRIPTION
**Pull request by guddyjack**

### CODE REVIEW

**Comment by guddyjack** [comment]
{
    "name": "Education(LEDU) Token",
    "website": "https://ledu.educationecosystem.com/",
    "description": "Education Ecosystem is a decentralized ecosystem for learning about new technologies and trying them out hands-on using free practical projects.",
    "explorer": "https://bscscan.com/token/0x887D9c01FFf04022da9c6D64a65a481a9e5D1FCa",
    "type": "BEP20",
    "symbol": "LEDU",
    "decimals": 8,
    "status": "active",
    "id": "0x887D9c01FFf04022da9c6D64a65a481a9e5D1FCa"
 "links": [
        {
            "name": "twitter",
            "url": "https://twitter.com/eduecosystem1"
        },
        {
            "name": "telegram",
            "url": "https://t.me/ledu_token"
        },
        {
            "name": "telegram_news",
            "url": "https://t.me/ledu_channel"
        },
        {
            "name": "whitepaper",
            "url": "https://ledu.educationecosystem.com/documentation/"
        },
        {
            "name": "coinmarketcap",
            "url": "https://coinmarketcap.com/currencies/education-ecosystem/"
        },
        {
            "name": "coingecko",
            "url": "https://www.coingecko.com/en/coins/education-ecosystem/"
        }
    ],
    "tags": [
        "memes"
    ]
}

File: 0x887D9c01FFf04022da9c6D64a65a481a9e5D1FCa/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "name": "Education(LEDU) Token",
+    "website": "https://ledu.educationecosystem.com/",
+    "description": "Education Ecosystem is a decentralized ecosystem for learning about new technologies and trying them out hands-on using free practical projects.",
+    "explorer": "https://bscscan.com/token/0x887D9c01FFf04022da9c6D64a65a481a9e5D1FCa",
+    "type": "BEP20",
+    "symbol": "LEDU",
+    "decimals": 8,
+    "status": "active",
+    "id": "0x887D9c01FFf04022da9c6D64a65a481a9e5D1FCa"
+}
```
Comment (guddyjack) [inline 0x887D9c01FFf04022da9c6D64a65a481a9e5D1FCa/info.json:11]: "links": [
        {
            "name": "twitter",
            "url": "https://twitter.com/eduecosystem1"
        },
        {
            "name": "telegram",
            "url": "https://t.me/ledu_token"
        },
        {
            "name": "telegram_news",
            "url": "https://t.me/ledu_channel"
        },
        {
            "name": "whitepaper",
            "url": "https://ledu.educationecosystem.com/documentation/"
        },
        {
            "name": "coinmarketcap",
            "url": "https://coinmarketcap.com/currencies/education-ecosystem/"
        },
        {
            "name": "coingecko",
            "url": "https://www.coingecko.com/en/coins/education-ecosystem/"
        }
    ],
    "tags": [
        "memes"
    ]

**Review by guddyjack — COMMENTED** [review]
kindly update the correct json file and update the token logo

**Review by guddyjack — COMMENTED** [review]
also kindly change token type to BEP20, not BEP2. Thanks

**Review by guddyjack — COMMENTED** [review]
kindly update, the errors are fixed

**Review by guddyjack — COMMENTED** [review]
kindly update this changes

**Comment by guddyjack** [comment]
Number of holders : 15,312

**Comment by cryptomanz** [comment]
Please create new PR, follow doc https://developer.trustwallet.com/assets/requirements
You may want to try our Assets app in beta; go to https://assets.trustwallet.com, log on with Github, enter contract ID, logo and some additional info, and it will create a PR for you.
See https://developer.trustwallet.com/assets/repository_details

**Comment by guddyjack** [comment]
Please, I guess you didn't check through out bsc scan, we have met the
requirements

**Comment by cryptomanz** [comment]
You need create new PR with correct folder:
- This is a BEP20 token, correct folder is blockchains/smartchain/assets. Please correct.

**Comment by guddyjack** [comment]
thanks

**Comment by guddyjack** [comment]
kindly help close all other PR concerning this update

**Pull request closed without merging**